src/MyHealth/index.tsx

```diff
@@ -1,64 +1,106 @@
 import React, { FunctionComponent } from "react"
-import { View, StyleSheet } from "react-native"
+import { ScrollView, StyleSheet, View } from "react-native"
+import { useNavigation } from "@react-navigation/native"
 import { useTranslation } from "react-i18next"
 
-import { useStatusBarEffect } from "../navigation"
-import { Text, StatusBar } from "../components"
-import SymptomLog from "./SymptomLog"
+import { useSymptomLogContext } from "./SymptomLogContext"
+import { SymptomLogEntry } from "./symptoms"
+import { Text, StatusBar, Button } from "../components"
+import { useStatusBarEffect, MyHealthStackScreens } from "../navigation"
+import SymptomLogListItem from "./SymptomLogListItem"
 
-import { Typography, Colors, Spacing } from "../styles"
+import { Buttons, Typography, Colors, Outlines, Spacing } from "../styles"
 
-const MyHealthScreen: FunctionComponent = () => {
-  useStatusBarEffect("dark-content", Colors.secondary10)
+const SymptomLog: FunctionComponent = () => {
+  useStatusBarEffect("dark-content", Colors.primaryLightBackground)
   const { t } = useTranslation()
+  const { symptomLogEntries } = useSymptomLogContext()
+  const navigation = useNavigation()
+
+  const hasSymptomHistory = symptomLogEntries.length > 0
+
+  const handleOnPressLogSymptoms = () => {
+    navigation.navigate(MyHealthStackScreens.SelectSymptoms)
+  }
+
+  const NoSymptomHistory = () => {
+    return (
+      <Text style={style.noSymptomHistoryText}>
+        {t("symptom_checker.no_symptom_history")}
+      </Text>
+    )
+  }
+
+  const SymptomHistory = () => {
```

Comment:
I see we are not grouping entries now, but shouldn't we sort them to stay consistent in displaying them?

Reply:
Yes! great catch, we are not sorting anymore. We were thinking about grabbing that in a follow up pr, but it makes sense in this one i think.